api/src/legacy_soap_api/grantors/services/get_submission_list_expanded_response.py

```diff
@@ -83,31 +98,88 @@ def get_submission_list_expanded_response(
             )
         )
     )
+
     grants_gov_tracking_numbers = None
     cfda_numbers = None
     funding_opportunity_numbers = None
+    opportunity_ids = None
+    competition_ids = None
+    package_ids = None
+    status = None
+    simpler_status = None
+    submission_titles = None
     if get_submission_list_expanded_request.expanded_application_filter:
         if (
             submission_filters := get_submission_list_expanded_request.expanded_application_filter.filters
         ):
             grants_gov_tracking_numbers = submission_filters.get("GrantsGovTrackingNumber")
             cfda_numbers = submission_filters.get("CFDANumber")
             funding_opportunity_numbers = submission_filters.get("FundingOpportunityNumber")
+            opportunity_ids = submission_filters.get("OpportunityID")
+            competition_ids = submission_filters.get("CompetitionID")
+            package_ids = submission_filters.get("PackageID")
+            status = submission_filters.get("Status")
+            submission_titles = submission_filters.get("SubmissionTitle")
 
-    # TODO: the agency from the certificate needs to be added to the query here
-    if grants_gov_tracking_numbers:
-        stmt = stmt.where(
-            ApplicationSubmission.legacy_tracking_number.in_(grants_gov_tracking_numbers)
-        )
-    if cfda_numbers:
-        stmt = stmt.where(OpportunityAssistanceListing.assistance_listing_number.in_(cfda_numbers))
-    if funding_opportunity_numbers:
-        stmt = stmt.where(Opportunity.opportunity_number.in_(funding_opportunity_numbers))
+    # Unsupported but logged
+    if competition_ids:
+        logger.info(f"GetSubmissionListExpanded Filter: CompetitionIDs {competition_ids}")
+    if package_ids:
+        logger.info(f"GetSubmissionListExpanded Filter: PackageIDs {package_ids}")
+    if submission_titles:
+        logger.info(f"GetSubmissionListExpanded Filter: SubmissionTitles {submission_titles}")
+
+    simpler_status = None
+    submissions_query: Sequence[ApplicationSubmission] = []
+    # If more than one status in filter then just return nothing
+    if status and len(status) > 1:
+        submissions_query = []
+    else:
```

Comment:
To avoid some of this nesting, a suggestion. Make a function like:

```
get_application_submissions( ... ) -> Sequence[ApplicationSubmission]:
    if status and len(status) > 1:
        return []

    # the else statement here
```
Then we can simplify the `for submission in submissions` bit below.